### PR TITLE
Adds "re-invite" button in weekly digest email

### LINKED ISSUE #2210: Refactor inline styling into a <style> element for app/views/supervisor_mailer/weekly_digest.html.erb
**Description**
Refactor all inline css inside a `<style>` element

### PR DESCRIPTION
**Pull request by efgalvao**
### What github issue is this PR for, if any?
Resolves #2210

### What changed, and why?
Added a list to weekly digest if the supervisor has invited a volunteer and he not accepted it yet,  changed the resend invitation method so it could be called from the button in e-mail, also created a method to list all pending invitations for a specific supevisor.

### How will this affect user permissions?
It will not.

### How is this tested? (please write tests!) 💖💪
There are some tests that checks for the volunteer in the email if he not accepted and another one when the volunteer has accepted.

### Screenshots please :)
Screenshot from 2021-07-25 19-52-33

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
efgalvao

### CODE REVIEW

File: spec/mailers/supervisor_mailer_spec.rb [hunk 1]
```diff
@@ -61,6 +61,26 @@
       create(:case_assignment, casa_case: casa_case, volunteer: volunteer, active: false, updated_at: Date.today - 8.days)
       expect(mail.body.encoded).to_not match("Summary for #{volunteer.display_name}")
     end
+
+    context "when a supervisor has pending volunteer to accepts invitation" do
+      let!(:volunteer2) { create(:volunteer) }
+      let!(:invite) { volunteer2.invite!(supervisor) }
```
Comment (wenderjean) [inline spec/mailers/supervisor_mailer_spec.rb:67]: It doesn't seem you're using `invite`, I mean, it'd probably fit better into a `before` block.

File: config/routes.rb [hunk 2]
```diff
@@ -92,7 +92,7 @@
     member do
       patch :activate
       patch :deactivate
-      patch :resend_invitation
+      get :resend_invitation
```
Comment (compwron) [inline config/routes.rb:95]: I am very ambivalent about a modifying operation being a get. I would prefer patch or post. Why does this need to be a get? Does it have to do with calling it from an email?

**Review by compwron — APPROVED** [review]

File: config/routes.rb (hunk 2 above)
Comment (efgalvao) [reply config/routes.rb:95]: Exactly, can call a patch from email.

Comment (compwron) [reply config/routes.rb:95]: Gotcha gotcha
Maybe someday we will find a better way, but at least it's authenticated 😅

**Comment by compwron** [comment]
Very cool!! 🎉✨

File: config/routes.rb (hunk 2 above)
Comment (efgalvao) [reply config/routes.rb:95]: Exactly.

**Pull request merged**